tests/lxd/test_lxd_backend.cpp

```diff
@@ -628,7 +631,10 @@ TEST_F(LXDBackend, unsupported_suspend_throws)
 
     mp::LXDVirtualMachine machine{default_description, stub_monitor, mock_network_access_manager.get(), base_url};
 
-    EXPECT_THROW(machine.suspend(), std::runtime_error);
+    const auto throw_msg_matcher = mp::make_optional(StrEq("suspend is currently not supported"));
```

Comment:
Why `optional`?

Reply:
lol, actually, no reason other than I copied that from other code and didn't think about it :smiley:   Will fix.

Reply:
Ok, fixed this.